warehouse/config.py

```diff
@@ -238,6 +241,20 @@ def configure(settings=None):
         coercer=int,
         default=300,
     )
+    maybe_set(
+        settings,
+        "remember_device.days",
+        "REMEMBER_DEVICE_DAYS",
+        coercer=int,
+        default=30,
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏I like this even more, since then we don't even need to set a value unless we want to override the default of 30 days.